### ISSUE TITLE
add driver package for RealTek network card 

### ISSUE THREAD

**Issue by SZsmartest** [ni/linux#65]
for example: RTL8111/RTL8168

**Comment by gratian** [comment]
Can you provide more details on which product you are trying to use the RealTek network hardware with? It would help us determine which kernel branch to target for a fix.

**Comment by SZsmartest** [comment]
Thanks for asking, current already supported drivers for Intel,Broadcom, Mellanox's net card.
however, look at the market, most of product was produced with RelTek's net work chipset. so we have to complied the driver for them, what is why I suggest to add driver for RelTek.

I'm use RTL8168, I complied the ko files by myself. so just my suggestion,not a issue, sorry about that.

> Can you provide more details on which product you are trying to use the RealTek network hardware with? It would help us determine which kernel branch to target for a fix.

**Comment by gratian** [comment]
@SZsmartest I agree those are common chipsets and I'm in favor of adding support for them. I was just asking which NI products you are planning to use them with because the ARM/Zynq based products like cRIO-906x, myRIO, roboRIO use a different kernel branch vs. Intel based controllers. In other words I'm just trying to figure out which kernel branches to submit the changes in.

**Comment by wyongjian** [comment]
@SZsmartest  Can you describe how to compile the 8111/8168 driver? Thanks，I'm getting a GCC error while compiling

> Thanks for asking, current already supported drivers for Intel,Broadcom, Mellanox's net card. however, look at the market, most of product was produced with RelTek's net work chipset. so we have to complied the driver for them, what is why I suggest to add driver for RelTek.
> 
> I'm use RTL8168, I complied the ko files by myself. so just my suggestion,not a issue, sorry about that.
> 
> > Can you provide more details on which product you are trying to use the RealTek network hardware with? It would help us determine which kernel branch to target for a fix.

**Comment by gratian** [comment]
Closing this old issue. Feel free to re-open if needed.
